src/view/home.js

```diff
@@ -185,6 +186,9 @@ const headerStyles = StyleSheet.create({
   depositBtn: {
     marginLeft: 3,
   },
+  title: {
+    fontSize: 10,
+  },
```

Comment:
Is only the font size different? Not the font weight also?

Reply:
I'm not sure because the image isn't in invision afaict, but I tried with `fontWeight: 'lighter'` and it looked the same to me so I left it out.